internal/stringutil/split_test.go

```diff
@@ -45,27 +45,102 @@ func TestSplitQuoted(t *testing.T) {
 			want:  []string{`"a`, `b`, `c`, `d`},
 		},
 		{
-			// Unterminated quoted-run will cause quotes to be ignored from the start of the string.
 			input: `"a;b";"c;d`,
-			want:  []string{`"a`, `b"`, `"c`, `d`},
+			want:  []string{`"a;b"`, `"c`, `d`},
 		},
 		{
 			// Quotes must be escaped via RFC2047 encoding, not just a backslash.
 			// b through c below must not be treated as a single quoted-run.
```

Comment:
I think this comment is now out of date, as we are treating b-c as single quoted run.  I am a little concerned that we are changing something that is noted as "must" in the comment, but given mediatype tests are still passing, I guess we will roll with it.

Reply:
yeah, I was not sure about its original behavior, because it is possible to have `Content-Type: application/rtf; charset=utf-8; name="\"žába\".jpg"` (in [this test](https://github.com/jhillyerd/enmime/blob/49069813358fa4de47947106637d5eecdbe3ef8f/mediatype/mediatype_test.go#L356)), which means quotes can be escaped

Reply:
thanks, comment removed